app/models/goal.py

```diff
@@ -1,5 +1,13 @@
 from app import db
```

Comment:
Remember to make an empty init.py file in any package folder/subfolder. app has one, but we should have one here in the models folder as well.